.github/workflows/cron.yml

```diff
@@ -99,6 +99,73 @@ jobs:
         fail_ci_if_error: false
         file: ./coverage.xml
 
+  cron-pip:
+    # pip install monai[all] and use it to run unit tests
+    if: github.repository == 'Project-MONAI/MONAI'
+    strategy:
+      matrix:
+        container: ["pytorch:21.02", "pytorch:21.04"]  # 21.02 for backward comp.
+    container:
+      image: nvcr.io/nvidia/${{ matrix.container }}-py3  # testing with the latest pytorch base image
+      options: "--gpus all"
+    runs-on: [self-hosted, linux, x64, common]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install the dependencies
+        run: |
+          which python
+          python -m pip install --upgrade pip wheel twine
+          python -m pip uninstall -y torch torchvision
+          python -m pip list
+      - name: Run tests report coverage
+        run: |
+          pip uninstall monai
+          pip list | grep -iv monai
+          git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+          root_dir=$PWD
+          echo "$root_dir"
+          set -e
+
+          # build tar.gz and wheel
+          python setup.py check -m -s
+          python setup.py sdist bdist_wheel
+          python -m twine check dist/*
+
+          # move packages to a temp dir
+          tmp_dir=$(mktemp -d)
+          cp dist/monai* "$tmp_dir"
+          rm -r build dist monai.egg-info
+          cd "$tmp_dir"
+          ls -al
+
+          # install from tar.gz
+          name=$(ls *.tar.gz | head -n1)
```

Comment:
Sorry, where does the tar.gz come from?

Reply:
sure,created at line133, copied at line 138